### PR TITLE
[ACR]Add registry usages API and SKU update

### PR DESCRIPTION
**Pull request by djyou**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by djyou** [comment]
@dsgouda I noticed the warning on naming `usages` to `listUsages` because it returns a list. The exact name should be ok to us either way but many other services have used `usages`. What is the guideline for this? /cc @SajayAntony

**Comment by dsgouda** [comment]
@djyou for list operations, it is recommended you name the operationId as \*_list\*

File: arm-containerregistry/2017-06-01-preview/swagger/containerregistry.json
Comment (dsgouda) [inline arm-containerregistry/2017-06-01-preview/swagger/containerregistry.json]: nit: Feel free to mark the readOnly properties wherever applicable

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by dsgouda** [comment]
@djyou Please let me know when you would like me to merge this.

**Comment by salameer** [comment]
@djyou what's the ETA for the Merge?

**Comment by djyou** [comment]
Still waiting for ARM's response on this new API. We can merge as soon as it is approved.

**Comment by salameer** [comment]
Great! please let us know, and sorry for pushing on this, but we try our best to avoid having open PRs too long on this Repo :)

**Comment by ravbhatnagar** [comment]
Looks good. ARM sign off.

**Comment by djyou** [comment]
Thank you @ravbhatnagar.
@salameer @dsgouda Please merge.
/cc @SajayAntony

File: arm-containerregistry/2017-06-01-preview/swagger/containerregistry.json
Comment (dsgouda) [inline arm-containerregistry/2017-06-01-preview/swagger/containerregistry.json]: How large is this list? Consider using `x-ms-pageable` if the page is large enough (>50 entries)

**Review by dsgouda — CHANGES_REQUESTED** [review]
Have a minor doubt I need to clarify

File: arm-containerregistry/2017-06-01-preview/swagger/containerregistry.json
Comment (djyou) [inline arm-containerregistry/2017-06-01-preview/swagger/containerregistry.json]: 2 entries.

Comment (dsgouda) [inline arm-containerregistry/2017-06-01-preview/swagger/containerregistry.json]: Thanks, no issues here then.

**Review by dsgouda — APPROVED** [review]

**Pull request merged**
